packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol

```diff
@@ -76,6 +78,7 @@ contract OptimismMintableERC721Factory is OwnableUpgradeable {
 
         OptimismMintableERC721 localToken = new OptimismMintableERC721(
             bridge,
+            _remoteChainId,
```

Comment:
The remote chain id should be set as an immutable value as part of the constructor/initializer here